src/csv-parser-allowed-channels.js

```diff
@@ -0,0 +1,17 @@
+function parseAllowedChannels(channels) {
+  if (!channels) {
+    return [];
+  }
+
+  return [
+    ...channels
+      .split(';')
+      .filter(Boolean)
+      .map((channelData) => {
+        const [id, name] = channelData.split(',');
+        return { name: name, value: id };
+      }),
+  ];
+}
+
```

Comment:
you can return the array directly 😄 
```suggestion
  return channels
      .split(';')
      .filter(Boolean)
      .map((channelData) => {
        const [id, name] = channelData.split(',');
        return { name: name, value: id };
      });
}

```

Reply:
Completed